cairo/ethereum/cancun/state.cairo

```diff
@@ -978,7 +986,8 @@ func storage_roots{
 
     // Create a Mapping[Address, Bytes32] that will contain the storage root of each address's
     // storage trie
-    let (map_addr_storage_root_start) = default_dict_new(0);
+    tempvar EMPTY_ROOT_PTR = new Bytes32Struct(EMPTY_ROOT_LOW, EMPTY_ROOT_HIGH);
```

Comment:
not sure the impact is high nor if it's actually much better in terms of steps, but just to mention it, you could do it with a label and `get_label_location`

Reply:
done